### DISCUSSION TITLE
How to run Hyprland on startup Endeavour OS GNOME ?

### DISCUSSION THREAD

**Discussion by neronguyen** [end-4/dots-hyprland#410] [Q&A] [answered]
I just run the recommend install command and can start with Hyprland in terminal but it not full screen. I already put ```source ~/.config/zshrc.d/dots-hyprland.zsh``` into my bash but Hyprland not start with startup or let me choose in the DM. What should I do ?

**Comment by end-4** [comment]
you can add an entry yourself
create `/usr/share/wayland-sessions/hyprland.desktop` with the following contents:
```ini
[Desktop Entry]
Name=Hyprland
Comment=Tiling compositor with the looks
Exec=Hyprland
Type=Application
```

**Reply by neronguyen** [reply]
I already see in it my wayland-sessions, but in the DM I only see my username and enter password to enter GNOME again.

**Reply by end-4** [reply, marked as answer]
maybe this
https://wiki.archlinux.org/title/GDM#Wayland_and_the_proprietary_NVIDIA_driver

**Reply by neronguyen** [reply]
thanks i can finally make it
